### PR TITLE
Update Configs to add Chaining Tip

### PR DESCRIPTION
**Pull request by rcarteraz**
- Added to each config page a tip about chaining commands w/ an example specific to that config. 
- Various formatting fixes.
- Removed flasher from tabs where applicable for consistency and to reflect status of setting configs in flasher being removed (except `lora.region`)
- Added missing tables for a few CLI tabs.

### CODE REVIEW

**Review by sachaw — APPROVED** [review]

File: docs/configuration/device-config/device.mdx [hunk 1]
```diff
@@ -68,8 +67,28 @@ All device config options other than NTP Server are available on iOS, iPadOS and
...
 
+:::info
+
 All device config options are available in the python CLI. Example commands are below:
 
+:::
+
+| Setting                  | Acceptable Values                                  | Default  |
+|--------------------------|----------------------------------------------------|----------|
+| device.debug_log_enabled | `true`, `false`                                    | `false`  |
```
[the remark this answers is not in the document]
Comment (caveman99) [reply docs/configuration/device-config/device.mdx:78]: default is false. if you set this to true the log output will NOT be switched off if protobuf activity is detected.

[the remark this answers is not in the document]
Comment (caveman99) [reply docs/configuration/device-config/device.mdx:78]: Reference: https://github.com/meshtastic/firmware/blob/450e7362f63295853e21d5f82e711634425ecc2f/src/SerialConsole.cpp#L71-L74

[the remark this answers is not in the document]
Comment (caveman99) [reply docs/configuration/device-config/device.mdx:78]: Summary: description and default are correct like this :-)

File: docs/configuration/module-config/telemetry.mdx [hunk 2]
```diff
@@ -103,11 +103,11 @@ All telemetry module config options are available in the python CLI. Example com
 
 |                         Setting                         |  Acceptable Values  | Default |
 | :-----------------------------------------------------: | :-----------------: | :-----: |
-|         telemetry.device_update_interval         | `integer` (seconds) |   `900`   |
+|         telemetry.device_update_interval         | `integer` (seconds) |   Default `0` is 15 minutes.   |
```
Comment (caveman99) [inline docs/configuration/module-config/telemetry.mdx:106]: Rather put 900 seconds here or at least add it, superficial readers may be tempted to set it to 20 (minutes) when they really wanna set 1200 (seconds).

**Review by caveman99 — CHANGES_REQUESTED** [review]

**Review by sachaw — APPROVED** [review]

**Review by caveman99 — APPROVED** [review]

**Pull request merged**